pkg/sanity/sanity.go

```diff
@@ -449,3 +460,68 @@ func PseudoUUID() string {
 func UniqueString(prefix string) string {
 	return prefix + uniqueSuffix
 }
+
+// Return codes for CheckPath
```

Comment:
Please introduce a type (like `type PathKind string`) and use that to make the code more type safe.

Reply:
Done